test/setupAfterEnv.js

```diff
@@ -0,0 +1,5 @@
+import { cleanup } from '@testing-library/react/pure';
+
+afterEach(() => {
+  cleanup();
+});
```

Comment:
We sometimes reset all modules in a beforeEach an re-require. Therefore we can't use the main entrypoint of `@testing-library/react` since that entrypoint may setup `beforeAll` hooks. So we have to use the `pure` entrypoint and manually cleanup